arches_querysets/rest_framework/serializers.py

```diff
@@ -489,9 +493,21 @@ def create(self, validated_data):
         validated_data["__request"] = self.context["request"]
         validated_data["__as_representation"] = True
         with transaction.atomic():
+            self.create_resource_if_missing(validated_data, entry_node=qs._entry_node)
             created = super().create(validated_data)
         return created
 
+    def create_resource_if_missing(self, validated_data, entry_node):
+        if validated_data.get("resourceinstance"):
+            return
+        # Would like to do the following, but we don't yet have a ResourceTileTree.save()
+        # method handling a fast path for empty creates:
+        # ResourceSubclass = self.fields["resourceinstance"].queryset.model
+        # So hardcode the Resource(Proxy)Model for now.
+        instance = Resource(graph=entry_node.graph)
+        instance.save(request=validated_data["__request"])
```

Comment:
For example, I've already started down this path here of calling into the old Resource proxy model to exercise the logic that saves edit log entries as side effects, but I'm hoping to factor that out of the proxy models soon.

Reply:
Edit log side effects now also lifted up in https://github.com/archesproject/arches/pull/12323, will circle back once that's merged and guard this under a version check.

Reply:
Well, I might leave this for a bit, it would require a little more surgery since the `Resource.save()` implementation assumes all saves are creates for the purpose of edit log entries.